instrumentation/netty/netty-4.1/javaagent/src/main/resources/tested_versions/netty-4.1

```diff
@@ -0,0 +1 @@
+4.1.0.Final
```

Comment:
Lets tests this against all 4.1.* versions

Reply:
did you check if we can support 4.0.*?
and do you know which version the customer has?

Reply:
4.0 does not supported by us, as the task was to support SpringWebflux 6. Spring WebFlux 6 uses netty 4.1